README.md

```diff
@@ -1016,6 +1016,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rohan Hari](https://github.com/rohan-hari)
   - [Rohan Nair](https://github.com/rohan9454)
   - [Rohinish K Singh](https://github.com/rohinish404)
+  - [Rohit Kumar] (https://github.com/rohitverse)
```

Comment:
- [Rohit Kumar](https://github.com/rohitverse)

add like this

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix